test/connectivity/scripts/connectivityTest.sh

```diff
@@ -69,6 +69,7 @@ function prepare_test_from_artifacts() {
     sed -i -e "s@<MetricsCollector.MetricsEndpointsCSV>@$METRICS_ENDPOINTS_CSV@g" "$deployment_working_file"
```

Comment:
btw some of these metric parameters are required, right? if so, please check if it is provided.